dissect/target/plugins/os/windows/dpapi/types/dpapi.py

```diff
@@ -0,0 +1,67 @@
+import struct
+
+from typing import BinaryIO
+from dissect.target.plugins.os.windows.dpapi.types.masterkey import _CUUID, _RestOfData
+
+from dissect import cstruct as _cstruct
+
+_dpapi_def = """
+struct DPAPIBlob
+{
+    DWORD dwVersion;
+    UUID provider;
+    DWORD mkVersion;
+    UUID guid;
+    DWORD flags;
+    BYTES_WITH_LEN description;
```

Comment:
The more idiomatic cstruct way to write this would be:
```suggestion
    DWORD descriptionLength;
    char description[descriptionLength];
```